static_report/src/components/Serve/ServeIndex.tsx

```diff
@@ -0,0 +1,234 @@
+                        <LinkBox as={Tr} _hover={{ bg: 'blackAlpha.50' }}>
+                          <Td w={0}>
+                            <Checkbox
+                              value={item.name}
+                              size="lg"
+                              zIndex={2}
+                              isChecked={checkState.includes(item.name)}
+                              onChange={(event) => {
+                                onSelectChanged(
+                                  item.name,
+                                  event.target.checked,
+                                );
+                                forceUpdate();
+                              }}
+                            />
+                          </Td>
+                          <Td>
+                            <LinkOverlay
+                              href={`/single-run/${item.name}/`}
+                              isExternal={true}
+                            >
+                              {item.name}
+                            </LinkOverlay>
+                          </Td>
+                          <Td>
+                            {item.datasource.name}: {item.datasource.type}
+                          </Td>
+                          <Td>{formatReportTime(item.created_at)}</Td>
+                        </LinkBox>
+                      ))}
+                  </Tbody>
+                </Table>
+              </TableContainer>
+            </TabPanel>
+
+            <TabPanel>
+              <TableContainer>
+                <Table variant="simple">
+                  <Thead>
+                    <Tr>
+                      <Th>Name</Th>
+                      <Th colSpan={2}>Base</Th>
+                      <Th colSpan={2}>Target</Th>
+                    </Tr>
+                    <Tr>
+                      <Th />
+                      <Th>Data Source</Th>
+                      <Th>Generated At</Th>
+                      <Th>Data Source</Th>
+                      <Th>Generated At</Th>
+                    </Tr>
+                  </Thead>
+                  <Tbody>
+                    {data &&
+                      data.comparison.map((item) => (
+                        <LinkBox as={Tr} _hover={{ bg: 'blackAlpha.50' }}>
+                          <Td>
+                            <LinkOverlay
+                              href={`/comparison/${item.name}/`}
+                              isExternal={true}
+                            >
+                              {item.name}
+                            </LinkOverlay>
+                          </Td>
+                          <Td>
+                            {item.base.datasource.name}:{' '}
+                            {item.base.datasource.type}
+                          </Td>
+                          <Td>
+                            {item.base.created_at &&
+                              formatReportTime(item.base.created_at)}
```

Comment:
Store this as `const` before the render return block?

Reply:
fixed